### PR TITLE
run stubtest on python 3.10

### PR DESCRIPTION
**Pull request by Akuli**
To do:
- [x] enable `stubtest_unused.py`
- [x] figure out which `py310.txt` lines are linux-specific, put them to `linux-py310.txt`

### CODE REVIEW

**Comment by JelleZijlstra** [comment]
`3.10-dev` might work?

File: .github/workflows/stubtest-unused-whitelist.yml [hunk 1]
```diff
@@ -16,7 +16,7 @@ jobs:
     strategy:
       matrix:
         os: ["ubuntu-latest", "windows-latest"]
-        python-version: [3.6, 3.7, 3.8, 3.9]
+        python-version: [3.6, 3.7, 3.8, 3.9, '3.10']
```
Comment (Akuli) [inline .github/workflows/stubtest-unused-whitelist.yml:19]:
```suggestion
        python-version: [3.6, 3.7, 3.8, 3.9, '3.10-dev']
```

**Review by srittau — APPROVED** [review]
Thanks!

**Comment by hauntsaninja** [comment]
Thanks for doing this, but I think I might roll back this change for now:
- This is running with an older alpha, so is currently getting in the way of changes
- With 3.9, even post-beta, we ended up having to switch off stubtest in CI for a while because the level of breakage was too high
- This PR will likely also cause problems with unused, I imagine there are some things that will need to be removed from py3_common
- We also at some point need to move the positional-only test to use 3.10 (we only check positional-only compatibility on latest version)

**Pull request merged**